README.md

```diff
@@ -1,3 +1,4 @@
+[![Stories in Ready](https://badge.waffle.io/mesosphere/marathon.png?label=ready,gui&title=Ready)](https://waffle.io/mesosphere/marathon?label=gui)
```

Comment:
Nice, I did not know you could do that. I would probably add a little text. This can be a little confusing. I had to look a little for it in order to find and understand the button:
![](http://cl.ly/image/1h341H1x0u1L/Image%202015-07-10%20at%2011%3A04%3A15.png)

I was actually just thinking of a link to waffle, but I guess this works too, but what use does the reader have of seeing how many stories are in ready? Also, I am not quite sure what that means, when a story is in ready?